testbot.py

```diff
@@ -160,4 +168,56 @@ async def join_game(arguments: list, message: discord.message):
     logging.info(f'Assigned {message.author} to {game_name}')
     await message.channel.send(f'Have fun playing {game_name}, {message.author}!')
 
+
+async def format_poll(message: discord.message):
+    command = message.content.replace("pb poll", '').strip()
+
+    if "\"" not in command or ',' not in command:
+        logging.debug('No poll title or arguments were passed')
+        await message.channel.send("You can't just make a poll without a title or arguments! Please insert a poll "
+                                   "title that is surrounded in \"\" as well as arguments proceeded with commas.")
+        return
+
+    title = command[command.find("\"") + 1:command.rfind("\"")]
+    options = command.replace(f'\"{title}\"', '').strip().split(", ")
+    logging.info(command)
+    poll = {
+        "title": title,
+        "options": options
+    }
+    logging.info('title --' + poll.get("title"))
+    logging.info(poll.get("options"))
+    return poll
+
+
+async def make_poll(arguments: dict, message: discord.message):
+    if not arguments:
+        logging.debug('No arguments passed')
+        await message.channel.send("A poll has to have a title and options to vote on!")
+        return
+
+    options = arguments.get("options")
+    argument_length = len(options)
+
+    if argument_length > 8:
+        logging.debug('Too many arguments passed')
+        await message.channel.send("Too many poll options! The maximum amount of poll options you can add is 8!")
```

Comment:
Above you refer to them as too many arguments passed, but here you say too many poll options. I think you should stick to one convention; `poll_options`.

Reply:
Fixed to match debug message.